efel/pyfeatures/pyfeatures.py

```diff
@@ -424,6 +425,23 @@ def spikes_in_burst1_burstlast_diff():
     ])
 
 
+def phaseslope_max():
+    """Calculate the maximum phase slope"""
+
+    voltage = get_cpp_feature("voltage")
+    time = get_cpp_feature("time")
+    time = time[:len(voltage)]
+
+    from numpy import diff
+
+    phaseslope = diff(voltage) / diff(time)
+    phaseslope_max = max(phaseslope)
+
+    if phaseslope_max.size == 0:
```

Comment:
I think this check won't work as intended because phaseslope_max is a scalar. I think this check should be performed before computing the max

Reply:
Fixed with a try/except in latest commit

Reply:
Thanks! looks great! nice that you switched to numpy.max ;)